ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb

```diff
@@ -0,0 +1,1073 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    IS_GOOGLE_CLOUD_NOTEBOOK = os.path.exists("/opt/deeplearning/metadata/env_version")
</pre><p>Remove this line as it was defined before.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
I actually added this line intentionally as the previous definition was before we restart the kernal and would be lost once kernal restarted after installing required package.